test/lib.bash

```diff
@@ -77,7 +77,12 @@ function serverless_operator_e2e_tests {
   done
   kubeconfigs_str="$(array.join , "${kubeconfigs[@]}")"
 
-  go_test_e2e -failfast -tags=e2e -timeout=30m -parallel=1 ./test/e2e \
+  RUN_FLAGS="-failfast -timeout=30m -parallel=1"
+  if [ -n "${OPERATOR_TEST_FLAGS:-}" ]; then
+    RUN_FLAGS="${OPERATOR_TEST_FLAGS}"
+  fi
```

Comment:
To pass shellcheck it could be something like this:
```
RUN_FLAGS=(-failfast -timeout=30m -parallel=1)
if [ -n "${OPERATOR_TEST_FLAGS:-}" ]; then
  RUN_FLAGS=("${OPERATOR_TEST_FLAGS}")
fi
...
go_test_e2e -tags=e2e "${RUN_FLAGS[@]}" ...
```

Reply:
yes in previous pr i did something similar. https://github.com/openshift-knative/serverless-operator/pull/1963
but in the eventing repo i saw this changes and then modified the existing ones
https://github.com/openshift-knative/serverless-operator/pull/1963
ok i am changing

Reply:
Please check my updated example above. The shellcheck is very strict about how the params can be passed.

Reply:
AFAIK, in the eventing repo there's no shell check. So, it will allow for doing things that are not allowed by shellcheck.